package.json

```diff
@@ -13,6 +13,7 @@
   "author": "stylelint",
   "files": [
     "getOsEol.js",
+    "getTestRule.js",
     "jest-preset.js",
     "jest-setup.js"
```

Comment:
Should this file be called `jest.setup.js` to be consistent with how we're suggesting authors configure their Jest?